WEB/BE/services/user/index.js

```diff
@@ -0,0 +1,28 @@
+const model = require('@models/sequelizeWEB.js');
```

Comment:
userModel을 바로 불러올 수도 있을 것 같네요
```suggestion
const userModel = require('@models/sequelizeWEB.js').user;
```

Reply:
!! 이부분도 다른 파일은 수정을 못해서 이따 바꿔야지 하고 잊고있던건데 ㅎㅎㅎㅎ 감사합니다